src/plugins/intel_gna/transformations/pwl_approximation.cpp

```diff
@@ -0,0 +1,544 @@
+    segments.push_back({0, 0, std::numeric_limits<double>::infinity()});
+    return true;
+}
+
+template<>
+bool pwl_search<ngraph::opset8::Power>(const std::shared_ptr<ngraph::opset8::Power>& node,
+                                       const std::shared_ptr<ngraph::Node>& fake_quantize,
+                                       double allowed_err_pct,
+                                       double& err_pct,
+                                       std::vector<details::Pwl>& segments) {
+    auto constant = std::dynamic_pointer_cast<ngraph::opset8::Constant>(node->get_input_node_shared_ptr(1));
+    double exponent = 0;
+    if (!ngraph_util::get_constant_value(constant, exponent)) {
+        throw std::runtime_error("The unsupported type of element.");
+    }
+
+    return pwl_search_power(node, exponent, 1, 0, fake_quantize, allowed_err_pct, err_pct, segments);
+}
+
+template<>
+bool pwl_search<ngraph::op::PowerIE>(const std::shared_ptr<ngraph::op::PowerIE>& node,
+                                     const std::shared_ptr<ngraph::Node>& fake_quantize,
+                                     double allowed_err_pct,
+                                     double& err_pct,
+                                     std::vector<details::Pwl>& segments) {
+    auto power = std::dynamic_pointer_cast<ngraph::op::PowerIE>(node);
+    return pwl_search_power(node, power->power, power->scale, power->shift, fake_quantize, allowed_err_pct, err_pct, segments);
+}
+
+template<typename T>
+bool transform_to_pwl(
+    const std::shared_ptr<ngraph::Node>& fake_quantize,
+    const std::shared_ptr<T>& node,
+    double allowed_err_pct) {
+    double err_pct = 0;
+    std::vector<details::Pwl> segments;
+    if (!pwl_search<T>(node, fake_quantize, allowed_err_pct, err_pct, segments)) {
+        return false;
+    }
+
+    std::vector<double> m(segments.size() - 1);
+    std::vector<double> b(segments.size() - 1);
+    std::vector<double> alpha(segments.size());
+    for (size_t i = 0; i < segments.size() - 1; i++) {
+        m[i] = segments[i].m;
+        b[i] = segments[i].b;
+        alpha[i] = segments[i].alpha;
+    }
+    alpha[segments.size() - 1] = segments[segments.size() - 1].alpha;
+
+    auto m_constant = std::make_shared<ngraph::opset8::Constant>(ngraph::element::Type_t::f64,
+        ngraph::Shape{segments.size() - 1}, m);
+    m_constant->set_friendly_name("PWL slope");
+    auto b_constant = std::make_shared<ngraph::opset8::Constant>(ngraph::element::Type_t::f64,
+        ngraph::Shape{segments.size() - 1}, b);
+    b_constant->set_friendly_name("PWL offset");
+    auto alpha_constant = std::make_shared<ngraph::opset8::Constant>(ngraph::element::Type_t::f64,
+        ngraph::Shape{segments.size()}, alpha);
+    alpha_constant->set_friendly_name("PWL alpha");
+    auto pwl = std::make_shared<Pwl>(
+        fake_quantize ? fake_quantize : node->input_value(0),
+        m_constant, b_constant, alpha_constant);
+    pwl->set_base_node(node);
```

Comment:
Can we avoid keeping shared pointer to the base node somehow? Can we keep required information only?

Reply:
Discussed offline. Decided to keep base node.